test/parallel/test-net-listen-exclusive-random-ports.js

```diff
@@ -9,11 +9,11 @@ if (cluster.isMaster) {
   const worker1 = cluster.fork();
 
   worker1.on('message', function(port1) {
-    assert.strictEqual(port1, port1 | 0, 'first worker could not listen');
+    assert.strictEqual(port1, port1 | 0, `first worker could not listen on port ${port1}`);
     const worker2 = cluster.fork();
 
     worker2.on('message', function(port2) {
-      assert.strictEqual(port2, port2 | 0, 'second worker could not listen');
+      assert.strictEqual(port2, port2 | 0, `second worker could not listen on port ${port2}`);
```

Comment:
This will fail the linter, unfortunately. Same above.

You could instead realign like so:

```js
      assert.strictEqual(port2, port2 | 0,
                         `second worker could not listen on port ${port2}`);

Reply:
The alternative would be to just remove the message altogether and instead show the default error message. That would also be fine with me.

Reply:
I went ahead and removed it.